src/python/scripts/dx-clone-asset

```diff
@@ -89,18 +98,22 @@ def _clone_asset_into_region(region, record_name, asset_properties, asset_file_n
     # Now try to run the CLONE_ASSET_APP num_retries + 1 times.
     curr_try = 0
     while curr_try <= num_retries:
-        job = CLONE_ASSET_APP.run(
-            executable_input={
-                'asset_properties': json.dumps(asset_properties), 'url': url,
-                'record_name': record_name, 'asset_file_name': asset_file_name},
-            project=project_id)
-        print('{0}: {1}'.format(region, job.get_id()), file=sys.stderr)
+        cmd = ['dx', 'run', CLONE_ASSET_APP_NAME, '--project', project_id, '-iurl=' + url, '-irecord_name=' + record_name]
+        cmd += ['-iasset_file_name=' + asset_file_name, '-iasset_properties=' + json.dumps(asset_properties), '--brief']
+        job = subprocess.check_output(cmd).strip()
+        print('{0}: {1}'.format(region, job), file=sys.stderr)
         try:
-            job.wait_on_done()
-            q.put(region)
-            return {region: job.describe()['output']['asset_bundle']}
-        except DXJobFailureError:
+            cmd = 'dx wait {0} '.format(job)
+            subprocess.check_output(cmd, shell=True)
+        except subprocess.CalledProcessError as cpe:
+            print(repr(cpe))
             pass
+        cmd = 'dx describe {0} --json '.format(job)
+        job_desc = json.loads(subprocess.check_output(cmd, shell=True).strip())
```

Comment:
Minor: it may be more convenient at times to use dxpy directly so there's less dependency on strings from stdout, e.g. https://github.com/dnanexus/dx-cwl/blob/master/dx-cwl#L556 (though this is code that eventually gets compiled to scatter app code so could be harder to read).  Maybe a better example is here: https://github.com/dnanexus/dx_app_builder/blob/orodeh-app-scatter/app_scatter/code.py

Reply:
Unfortunately when using the `multiprocessing` map_async functionality with the `dxpy` package directly to make request (no subprocess), the `requests` package throws `requests.packages.urllib3.exceptions.SSLError` error.